brave/brave-ios
# Fix #2604: Add default icon for coinmarketcap.

## Summary of Changes


This pull request fixes #2604

## Submitter Checklist:


## Test Plan:

1. Install the app
2. Verify the icon for coinmarketcap.com is showing correctly, no monogram letter is used.

## Screenshots:


<img width="648" alt="Zrzut ekranu 2020-06-11 o 18 22 59" src="https://user-images.githubusercontent.com/6950387/84413929-94420e80-ac11-11ea-866a-7f0e7acefa62.png">


## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
